ocp_resources/resource.py

```diff
@@ -1266,6 +1272,13 @@ def wait_for_condition(
             if sample:
                 for cond in sample.get("status", {}).get("conditions", []):
                     actual_condition = {"type": cond["type"], "status": cond["status"]}
+
+                    if stop_condition:
+                        if actual_condition == {"type": stop_condition, "status": stop_status}:
```

Comment:
for clarify, as suggested by code rabbit, please combine the 2 conditions - see https://github.com/RedHatQE/openshift-python-wrapper/pull/2585#pullrequestreview-3483133163

Also please see https://github.com/RedHatQE/openshift-python-wrapper/pull/2585#pullrequestreview-3483564176 - please update docstring to reflect that "message" is ignored in this flow